### PR TITLE
Submission page using form components and reading from store

### PR DESCRIPTION
**Pull request by TimCsaky**
Using Vue store and form components on Admin > submission review page
i should have done this last week!

### CODE REVIEW

**Comment by TimCsaky** [comment]
Let me know if anyone wants the submitted date and confirmation id on this page as well. 
it's currently only on the submissions table.

File: app/frontend/src/views/admin/Submission.vue [hunk 1]
```diff
@@ -1,608 +1,94 @@
...
-                  <td>Agreed that farm will be subject to a site inspection</td>
-                  <td>
-                    <v-icon color="green" v-if="ipcPlanData.ipcPlan.agreeToInspection">check</v-icon>
-                    <v-icon color="red" v-else>close</v-icon>
-                  </td>
-                </tr>
-              </tbody>
-            </template>
-          </v-simple-table>
+    <BaseSecure admin>
```
Comment (parc-jason) [inline app/frontend/src/views/admin/Submission.vue:3]: may want @jujaga to review that security? i thought he changed it....

**Comment by loneil** [comment]
> Let me know if anyone wants the submitted date and confirmation id on this page as well.
> it's currently only on the submissions table.

@TimCsaky yeah I think it's a good idea to have a header of the submitted date, confirmation id, (maybe business name big at the top if it's not already) and the print pdf button.

**Comment by loneil** [comment]
@TimCsaky can probably close this as you mentioned in the other PR and delete the openshift PR.

**Comment by TimCsaky** [comment]
this stuff was copied over to another pr

**Pull request closed without merging**